src/Relude/File.hs

```diff
@@ -50,19 +50,38 @@ import qualified Data.Text.Lazy.IO as LT
 -- Text
 ----------------------------------------------------------------------------
 
--- | Lifted version of 'T.readFile'.
+{- | Lifted version of 'T.readFile'.
+
+'T.readFile' is a function usually used when you want to read files inside IO monad,
+but this 'readFileText' is lifted which means that you can use it even inside various
+monad transformers without adding liftIO call explicitly.
+
+-}
 readFileText :: MonadIO m => FilePath -> m Text
 readFileText = liftIO . T.readFile
 {-# SPECIALIZE readFileText :: FilePath -> IO Text #-}
 {-# INLINE     readFileText #-}
 
--- | Lifted version of 'T.writeFile'.
+{- | Lifted version of 'T.writeFile'.
+
+'T.writeFile' is a function usually used when you want to readwrite files inside IO monad,
+but this 'writeFileText' is lifted which means that you can use it even inside various
+monad transformers without adding liftIO call explicitly.
```

Comment:
Hmm, this is some documentation duplication :thinking: Let's move it to top-level docs. And can even move to `Relude.Lifted` top-level docs. To cover all the modules at once.

Reply:
yeah I was unsure about that kind of looked a little silly, will make it top level rather than just slight variations